src/paps.1

```diff
@@ -250,17 +250,20 @@ Portable Document Format
 Scalable Vector Graphics
 .RE
 .TP
+.B Postscript points
+Each postscript point equals to 1/72 of an inch. 36 points are 1/2 of an inch.
+.TP
 .B \-\-bottom-margin=bm
-Set bottom margin in postscript points (1/72 inch). Default is 36.
+Set bottom margin. Default is 36 postscript points.
```

Comment:
I'll accept your changes, though my version is more exact. Stating that the default is 36 postscript units, does not specify in what unit to specify the bottom margin. The same is true for the other units and the gutter margin.